cmd/amppkg/main.go

```diff
@@ -90,7 +91,7 @@ func main() {
 	if certs == nil || len(certs) == 0 {
 		die(fmt.Sprintf("no cert found in %s", config.CertFile))
 	}
-	if !*flagDevelopment && !util.CanSignHttpExchanges(certs[0]) {
+	if (!*flagDevelopment && !*flagInvalidCert) && !util.CanSignHttpExchanges(certs[0]) {
```

Comment:
I slightly prefer the De Morgan equivalent `!(*flagDevelopment || *flagInvalidCert)` but am amenable to either.

Reply:
No problem. Does this apply to line 130 as well? e.g.
`/*requireHeaders=*/!(*flagDevelopment || *flagInvalidCert)`

Reply:
Yeah.

Reply:
Resolved in commit f78e86e